endpoints/api/billing.py

```diff
@@ -20,6 +20,7 @@
 from endpoints.api import (
     ApiResource,
     abort,
+    allow_if_global_readonly_superuser,
```

Comment:
this has been added to a lot of endpoints that are only used on quay.io, for example getting the credit card or subscription. The permissions add can be removed on those endpoints

Reply:
Reverted the changes to the billing endpoints